src/main/java/hudson/plugins/git/GitTagAction.java

```diff
@@ -185,15 +185,10 @@ void scheduleTagCreation(Map<String, String> newTags, String comment) throws IOE
      */
     public final class TagWorkerThread extends TaskThread {
         private final Map<String, String> tagSet;
-        /**
-         * If the user provided a separate credential, this object represents that.
-         */
-        private final String comment;
 
-        public TagWorkerThread(Map<String, String> tagSet,String comment) {
+        public TagWorkerThread(Map<String, String> tagSet,String ignoredComment) {
```

Comment:
I guess `ignoredComment` is still there just to keep the binary compatibility, right?

Reply:
That's correct.  I didn't want to risk a signature change when a change of parameter name would be enough.